style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Space+Mono&display=swap');
+
+.headline {
+  background-image: url(./images/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
## [OPTIONAL]
- [ ] CONGRATULATIONS :clap: I  can see you follow the requirement of the first reviewer by putting the hamburger button in the `<nav>` tag and not in a `<a>` tag. However, you still need to stick to the design as much as possible as she said.
- Try to compare the original design on the left side and yours on the right side. Then improve your code by removing the `overflow-y: scroll` property or changing it with `overflow-y: hidden`. I think it would be nice also if you follow the same `font-family`, `font-weight`, and the `font-size` in the original design as the requirements said.
![Screenshot from 2021-11-26 11-50-09](https://user-images.githubusercontent.com/65068771/143563294-8e9d5bf6-ccd8-40eb-99d7-37c12f72e60f.png)